index.js

```diff
@@ -211,15 +240,16 @@ async function installBundler(bundlerVersionInput, platform, rubyPrefix, engine,
   }
 }
 
-async function bundleInstall(platform, engine, version) {
-  if (!fs.existsSync('Gemfile')) {
-    console.log('No Gemfile, skipping "bundle install" and caching')
-    return
+async function bundleInstall(gemfilePath, lockPath, platform, engine, version) {
+  if (!fs.existsSync(gemfilePath)) {
```

Comment:
This was already tested above, so we can just do a `!= null` check here.
Also `fs.existsSync()` probably does not like a `null` argument.
I can do it.

Reply:
Actually
```
> fs.existsSync(null)
false
```
because JavaScript.